packages/driver/cypress/e2e/commands/actions/type_special_chars.cy.js

```diff
@@ -75,18 +75,17 @@ describe('src/cy/commands/actions/type - #type special chars', () => {
   })
 
   context('parseSpecialCharSequences: false', () => {
-    it('types special character sequences literally', (done) => {
```

Comment:
All skips and fixes here are under the umbrella issue [23160](https://github.com/cypress-io/cypress/issues/23160)